controls/overlay.rb

```diff
@@ -3,13 +3,11 @@
 include_controls 'pgstigcheck-inspec' do

```

Comment:
@ejaronne This file seems to have a lot of controls that have the same metadata as the baseline, do you know why the NIST tags are listed in the overlay if they are not being changed?